systemd/ceph-osd@.service

```diff
@@ -19,6 +19,7 @@ TasksMax=infinity
 Restart=on-failure
 StartLimitInterval=30min
 StartLimitBurst=3
+RestartSec=5min
```

Comment:
I see the default value is 100ms (https://www.freedesktop.org/software/systemd/man/systemd.service.html) 
and the "osd_heartbeat_grace" is 20s by default, may be 20s or around is an option